src/code/components/chromosome.js

```diff
@@ -90,7 +82,7 @@ const ChromosomeView = ({chromosome, org, ChromosomeImageClass=ChromosomeImageVi
     <div className="geniblocks chromosome-container" onClick={ handleSelect } >
       <div className={ containerClass }>
         <div className="chromosome-allele-container" id={chromId} style={displayStyle}>
```

Comment:
I see what you mean- looking again, I see I left out the `orgName` in the `id`. I agree with your point above, that IDs have a lot of information shoved into them and it's not clear how it's all used. For now, I'll pull what I can into utility functions as you suggest so it's easier to address in the future, which feels like a step in the right direction.